lib/main.dart

```diff
@@ -37,7 +37,7 @@ class MyApp extends StatelessWidget {
         ),
         home: authProvider.isSignedIn() ? AuthScreen() : HomeScreen(),
         routes: {
-          "/home": (context) => HomeScreen(),
+          "/home": (context) => HomeScreen(cameras),
```

Comment:
you are right, i will change it. first i will collect all the permissions into the provider and then pass it through the HomeScreen widget. i will keep it like this in this PR but this will be me next task.